From 723c6207a40f7b1d1e677ffc8fd76db31ee78329 Mon Sep 17 00:00:00 2001
From: Ionut Nechita <ionut_n2001@yahoo.com>
Date: Sun, 13 Mar 2022 13:50:24 +0200
Subject: [PATCH 19/52] SUNLIGHT: Change default compiler optimization level

Description:
 - Optimize more for performance (-O3)
 - Default CC_OPTIMIZE_FOR_PERFORMANCE_O3
 - Include all platforms

Bug: N/A
Change-Id: I1f750eadafb98f3d6f68714d978a6bed0dae38d1
Signed-off-by: Ionut Nechita <ionut_n2001@yahoo.com>
---
 init/Kconfig | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/init/Kconfig b/init/Kconfig
index 4f94f87dcfca..abc38f2bbb32 100644
--- a/init/Kconfig
+++ b/init/Kconfig
@@ -1363,7 +1363,7 @@ config BOOT_CONFIG
 
 choice
 	prompt "Compiler optimization level"
-	default CC_OPTIMIZE_FOR_PERFORMANCE
+	default CC_OPTIMIZE_FOR_PERFORMANCE_O3
 
 config CC_OPTIMIZE_FOR_PERFORMANCE
 	bool "Optimize for performance (-O2)"
@@ -1374,7 +1374,6 @@ config CC_OPTIMIZE_FOR_PERFORMANCE
 
 config CC_OPTIMIZE_FOR_PERFORMANCE_O3
 	bool "Optimize more for performance (-O3)"
-	depends on ARC
 	help
 	  Choosing this option will pass "-O3" to your compiler to optimize
 	  the kernel yet more for performance.
-- 
2.35.1

